tests/test_recommendation_manager.py

```diff
@@ -1,7 +1,15 @@
 from taar.profile_fetcher import ProfileFetcher
 from taar.recommenders import RecommendationManager
 from taar.recommenders.base_recommender import BaseRecommender
-from taar import hbase_client
+# TODO: pull the MockProfileController out into a testing base module
```

Comment:
Let's do this, please, or file an issue to keep track of it

Reply:
Urgh.  The tests are currently living outside of the taar namespace so we can't easily extract the class into a separate module.  I just dropped the TODO and left the MockProfileController in the `tests/test_recommendation_manager.py`.